src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/image/index.js

```diff
@@ -0,0 +1,97 @@
+import { getImageParts } from './helpers';
+
+const captionBlock = ({ caption }) => {
+  if (!caption) return null;
+
+  return {
+    model: {
+      blocks: [
+        {
+          model: {
+            blocks: [
+              {
+                model: {
+                  blocks: [
+                    {
+                      model: {
+                        attributes: [],
+                        text: caption,
```

Comment:
Can we check if a caption can contain double quote marks `"` ... if it's `\"` then nothing needs to happen, if it's `&quot;` then it can use the helper added in https://github.com/bbc/simorgh/pull/4401 on this line and line 23 as `escapeDoubleQuotes(caption)`

Reply:
`/pidgin/world-23252817` has 
```
"caption": "Some people went to the NTA awards recently. This has \"quotes\" in it.",
```
 :)

Reply:
![](https://media.giphy.com/media/PhKhSXofSAm3e/giphy.gif)